BD-1364: Fix typo in Analytics overview doc (#3491)
braze-inc/braze-docs: _docs/_developer_guide/platform_wide/analytics_overview.md

Before completing your Braze implementation, ensure that you have a conversation between your marketing team and your development team regarding your marketing goals. When deciding what you want to track, and how you want to track it with Braze, it's useful to consider these goals and work backward from there. Reference our case of a [Taxi/Ride-Sharing App][16] at the end of this guide for an example of this process.

> This best practice guide will help you to understand exactly what Braze considers to be a custom event and a custom attribute.

Reviewer: This kinda implies that something can be a custom event and a custom attribute at the same time 🤔  I'd suggest a rework such as:
```suggestion
This best practice guide will help you understand the difference between a custom event and a custom attribute in Braze.
```
